.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,89 @@
+name: Bug Report
+description: Found something you weren't expecting?  Report it here!
+body:
+- type: markdown
+  attributes:
+    value: |
+      NOTE: If your issue is a security concern, please send an email to security@gitea.io instead of opening a public issue.
+- type: markdown
+  attributes:
+    value: |
+      1. Please speak English, this is the language all maintainers can speak and write.
+      2. Please ask questions or configuration/deploy problems on our Discord
+         server (https://discord.gg/gitea) or forum (https://discourse.gitea.io).
+      3. Please take a moment to check that your issue doesn't already exist.
+      4. Make sure it's not mentioned in the FAQ (https://docs.gitea.io/en-us/faq)
```

Comment:
Should this still be here now that we have it already on the new issue landing page with the config.yaml?

Reply:
Again, keeps it present and current so that they can see it.  It never hurts to remind them multiple times.